weave-js/src/components/PagePanelComponents/Home/Browse3/refs.ts

```diff
@@ -0,0 +1,56 @@
+import {
```

Comment:
much of this file already existed - moved from `weave-js/util/refs.ts` to here as this is scoped to "New Weave" (aka Browse3)